src/extension/tab/tabRelay.js

```diff
@@ -0,0 +1,45 @@
+import Relay from '../../Relay';
+
+async function requestId() {
+  const promise = new Promise((resolve) => {
+    chrome.runtime.sendMessage({ message: 'request' }, async function(response) {
+      resolve(response);
+    });
+  });
+
+  return await promise;
+}
```

Comment:
```suggestion
function requestId() {
  return new Promise((resolve) => {
    chrome.runtime.sendMessage({ message: 'request' }, response => resolve(response));
  });
}
```
No need for `async`/`await` when you can just return a `Promise`.